Project.toml

```
@@ -49,6 +51,7 @@ KrylovKitExt = "KrylovKit"
 Arrow = "1.5, 2"
 BSON = "0.3"
 Combinatorics = "1"
+CommonSolve = "0.2.4"
```

```suggestion
CommonSolve = "0.2"
```
Is there a need to depend on the minor version?

---

Yes. `step!` was introduced in this minor version.